openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -3625,9 +3625,12 @@ def create_force(self, system, topology, **kwargs):
                 #                             "must be provided")
             else:
                 force.setNonbondedMethod(openmm.NonbondedForce.LJPME)
-                force.setCutoffDistance(9.0 * unit.angstrom)
+                force.setCutoffDistance(self.cutoff)
                 force.setEwaldErrorTolerance(1.0e-4)
 
+                force.setSwitchingDistance(self.switch_width)
```

Comment:
@SimonBoothroyd I think this is the only substantive question remaining before I can make this merge-ready. The use of "width" makes me think that this is the more correct interpretation, since the switching function applies _between_ this distance and the cutoff. But since you're more on the physics side, I'll defer this decision to you. 


```suggestion
                force.setSwitchingDistance(self.cutoff - self.switch_width)
```

Reply:
The width would usually refer to the distance over which it is tapered, e.g. an angstrom or two or some such (e.g. switched between 8 and 10 angstroms) if that helps answer this question.

Reply:
@j-wags is correct here: For OpenMM, you need to use `force.setSwitchingDistance(self.cutoff - self.switch_width)`.